src/types/Patterns.ts

```diff
@@ -88,6 +88,8 @@ export const Patterns = {
   connectors: {
     NAME: /^[^-\s()'"#@.&](?!.*[%&?,';:!\s-]{2}).{1,19}$/,
     COUNTRY: /^[A-Z]{2}$/,
+    TECHNICAL_USER_NAME: /^[A-Za-z]+[a-zA-Z0-9 =.!&#'"()]{2,80}$/,
+    TECHNICAL_USER_DESCRIPTION: /^[A-Za-z]+[a-zA-Z0-9 =.!&#'"()]{2,120}$/,
```

Comment:
The character selection looks really random - why are '!', '#' and '&' allowed but not '?', '-' or '_' ...? No comma in a description seems quite restrictive. Also NEVER use '+' or '*' in an input pattern because this means it can have infinite length which is not good in any scenario.